lib/cadence/execution_options.rb

```diff
@@ -4,7 +4,7 @@ module Cadence
   class ExecutionOptions
     attr_reader :name, :domain, :task_list, :retry_policy, :timeouts, :headers
 
-    def initialize(object, options = {})
+    def initialize(object, options, defaults = nil)
```

Comment:
Can you add a comment as to what the shape of `defaults` is and what to pass in?

Reply:
It is an instance of `Cadence::Configuration::Execution`, but I see your point, this is immediately visible and can cause confusion. Agree that we need to address it